test/cookie.ts

```diff
@@ -1,10 +1,15 @@
-import test from 'ava';
+import test, {ExecutionContext} from 'ava';
+// @ts-ignore
 import PNG from 'png-js';
```

Comment:
I couldn't declare the module with  `.d.ts` file because `ts-node` is throwing the below error.
```
TypeError: Unable to require `.d.ts` file.
  This is usually the result of a faulty configuration or import. Make sure there is a `.js`, `.json` or another executable extension and loader (attached before `ts-node`) available alongside `png-js.d.ts`
```
I would love to know if this can be solved in a better way.

Reply:
https://github.com/TypeStrong/ts-node/issues/615

Reply:
I have tried all mentioned solutions but none worked.

Reply:
I will try to submit `png-js` to DefinitelyTyped.